schemas/SDMXStructureOrganisation.xsd

```diff
@@ -296,6 +309,19 @@
 		</xs:complexContent>
 	</xs:complexType>
 
+	<xs:complexType name="MetadataProviderType">
+		<xs:annotation>
+			<xs:documentation>MetadataProviderType defines the structure of a metadata provider description. The contacts defined for the organisation are specific to the metadata provider role the organisation is serving.</xs:documentation>
+		</xs:annotation>
+		<xs:complexContent>
+			<xs:extension base="MetadataProviderBaseType">
+				<xs:sequence>
+					<xs:element name="MetadataConstraints" type="common:MetadataConstraintAttachmentType" minOccurs="0"/>

```

Comment:
Should be `DataConstraintAttachmentType` instead.

Reply:
I guess this should actually be removed - we last discussed Metadata Provider and Data Provider no longer reference a Constraint because its only purpose was for the advanced release calendar?

Reply:
I have reverted this change so Metadata Provider no longer references a constraint to be in line with the documentation